### ISSUE TITLE
Can't converge when switching to MXnet backend

### ISSUE THREAD

**Issue by Pavel-Akapian** [awslabs/keras-apache-mxnet#108]
My training keras-tensorflow pipeline works fine, but when switching to keras-mxnet got warning: 
`Optimizer created manually outside Module but rescale_grad is not normalized to 1.0/batch_size/num_workers (1.0 vs. 0.125). Is this intended?
  force_init=force_init)`. 
Can it be the reason why my network can't converge? And what are possible mistakes that produce rescale_grad warning?
I'm not using multi-gpu.

**Comment by roywei** [comment]
Hi, not necessary, could you share more details? what model and optimizer are you using? A minimum reproducible code would help us debug.
Thanks!

**Comment by kalyc** [comment]
Hi @Pavel-Akapian 
Closing this issue at the moment. Please feel free to comment and re-open the issue and add further details.
